src/fromager/overrides.py

```diff
@@ -37,18 +37,23 @@ def _die_on_plugin_load_failure(
     mgr: extension.ExtensionManager,
     ep: extension.Extension,
     err: Exception,
-):
+) -> None:
     raise RuntimeError(f"failed to load overrides for {ep.name}") from err
 
 
-def find_and_invoke(distname: str, method: str, default_fn: typing.Callable, **kwargs):
+def find_and_invoke(
+    distname: str,
+    method: str,
+    default_fn: typing.Callable,
+    **kwargs: dict[str, typing.Any],
+) -> typing.Any:
     fn = find_override_method(distname, method)
     if not fn:
         fn = default_fn
     return invoke(fn, **kwargs)
 
 
-def invoke(fn: typing.Callable, **kwargs):
+def invoke(fn: typing.Callable, **kwargs: dict[str, typing.Any]) -> typing.Any:
```

Comment:
`*args` and `**kwargs` are typically annotated by member type, https://peps.python.org/pep-0484/#arbitrary-argument-lists-and-default-argument-values
```suggestion
def invoke(fn: typing.Callable, **kwargs: typing.Any) -> typing.Any:
```

Reply:
I have added another TODO item to the ticket to track the change.